venues/ICLR.cc/2019/Conference/python/notes.py

```diff
@@ -52,12 +52,15 @@ def freeze_and_post(client, note):
 
 def post_blind_note(client, original_note):
     blind_note = client.post_note(create_blind_note(original_note))
+    bibtex_entry = getBibtex(client, blind_note)
     paper_group_id = iclr19.CONFERENCE_ID + "/Paper{}".format(blind_note.number)
     author_group_id = iclr19.CONFERENCE_ID + "/Paper{}/Authors".format(blind_note.number)
 
-    # Update Blind note's contents, repost the updated blind note, and freeze and post the original note
-    blind_note.content["authorids"] = [author_group_id]
-    blind_note.content["_bibtex"] = getBibtex(client, blind_note)
+    blind_note.content = {
```

Comment:
This is exactly what we did last year!

Reply:
And I also must say that this change was in the Mohit script too. Again, we need to be very careful when we refactor code because we don't have automatic tests. Using code from previous conferences is the most accurate to do, we know that at least it worked before.